### PR TITLE
My app

### PR DESCRIPTION
**Pull request by Trust-007**
# Tasks Completed ✔️ 
- Add the following validations:
> For the User model:
> Name must not be blank.
> PostsCounter must be an integer greater than or equal to zero.
> For the Post model:
> Title must not be blank.
> Title must not exceed 250 characters.
> CommentsCounter must be an integer greater than or equal to zero.
> LikesCounter must be an integer greater than or equal to zero.

- Add unit specs for all of your models' methods and validations

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -51,6 +51,7 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rspec-rails'
```
Comment (rustam-tolipov) [inline Gemfile:54]: [OPTIONAL] Kindly personalize your MIT licese to your project. Thank you :)

File: spec/model/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  it 'post counter should be equal or great than 0' do
+    subject.posts_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'Subject should be valid' do
+    expect(subject).to be_valid
+  end
+
+  it 'Returns 3 most recent post' do
```
Comment (rustam-tolipov) [inline spec/model/user_spec.rb]: - [ ] At that point we have one failing test. Please fix the issue and resubmit it again. Also, if the app is working on your side try to reset your database with `rails db:reset` and run the test again. After that if the tests are working smoothly you are free to leave your screenshot below. Thank you. This is the issue we are facing:

Screenshot 2023-02-24 at 18 05 52

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Trust-007,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_I like the way you write the test description_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @Trust-007 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**